app/uk/gov/hmrc/exports/scheduler/jobs/PurgeAncientSubmissionsJob.scala

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.scheduler.jobs
+
+import org.bson.Document
+import org.bson.json.{JsonMode, JsonWriterSettings}
+import org.mongodb.scala.model.Filters._
+import play.api.Logging
+import uk.gov.hmrc.mongo.play.json.Codecs
+import play.api.libs.json.{Format, Json}
+import uk.gov.hmrc.exports.config.AppConfig
+import uk.gov.hmrc.exports.models.declaration.ExportsDeclaration
+import uk.gov.hmrc.exports.models.declaration.notifications.{ParsedNotification, UnparsedNotification}
+import uk.gov.hmrc.exports.repositories._
+import uk.gov.hmrc.exports.models.declaration.submissions.{Submission, SubmissionRequest}
+import uk.gov.hmrc.exports.mongo.ExportsClient
+
+import java.time._
+import javax.inject.{Inject, Singleton}
+import scala.collection.convert.DecorateAsScala
+import scala.concurrent.duration._
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class PurgeAncientSubmissionsJob @Inject() (
+  val appConfig: AppConfig,
+  submissionRepository: SubmissionRepository,
+  declarationRepository: DeclarationRepository,
+  unparsedNotificationRepository: UnparsedNotificationWorkItemRepository,
+  parsedNotificationRepository: ParsedNotificationRepository,
+  transactionalOps: PurgeSubmissionsTransactionalOps
+)(implicit ec: ExecutionContext)
+    extends ScheduledJob with ExportsClient with Logging with DecorateAsScala {
```

Comment:
You cannot use `ExportsClient` here. You are mixing Migrations' logic and Scheduler's logic.
The Migration package is a module injected before any other module. In addition, it uses the Mongo driver directly, whereas the Scheduler uses the facade provided by the hmrc-mongo library.